go.mod

```diff
@@ -1,19 +1,53 @@
 module github.com/cert-manager/csi-lib
 
-go 1.16
+go 1.17
 
 require (
 	github.com/container-storage-interface/spec v1.4.0
-	github.com/go-logr/logr v0.4.0
-	github.com/golang/protobuf v1.5.2 // indirect
-	github.com/jetstack/cert-manager v1.3.1
+	github.com/go-logr/logr v1.2.0
+	github.com/jetstack/cert-manager v1.7.0-beta.0
```

Comment:
/hold until we have the stable version out 😊

Reply:
Done.